rust/src/services/messages/state_machine.rs

```diff
@@ -23,9 +23,11 @@ use crate::{
     PetError,
 };
 
-/// The request type handled by [`StateMachineService`]
+/// [`StateMachineService`] request type
 #[derive(Debug, From)]
 pub struct StateMachineRequest(MessageOwned);
+
+/// `[StateMachineService`] response type
```

Comment:
``[`StateMachineService`]`` is a dead link. this should be easily fixable by adding the following line:
```rust
/// [`StateMachineService`]: struct.StateMachineService.html
```

Reply:
That shouldn't be broken though with the nightly doc feature... I'll look into it.

Reply:
huh it's just a typo actually. I put the backtick outside of the bracket.